pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -826,3 +826,67 @@ def test_complete_in_alter_keyspace(self):
         self.trycompletions('ALTER KEYSPACE system_trac', "es WITH replication = {'class': '")
         self.trycompletions("ALTER KEYSPACE system_traces WITH replication = {'class': '", '',
                             choices=['NetworkTopologyStrategy', 'SimpleStrategy'])
+
+    def test_complete_in_grant(self):
+        self.trycompletions(
+            "GR",
+            immediate='ANT ')
```

Comment:
```suggestion
        self.trycompletions("GR",
                                          immediate='ANT ')
```
I would probably keep the formatting for this and all following invocations similar to the one used in the other tests in the same class.